js/apps/account-ui/src/groups/Groups.tsx

```diff
@@ -69,7 +69,10 @@ const Groups = () => {
             />
           </DataListItemRow>
         </DataListItem>
-        <DataListItem id="groups-list-header" aria-labelledby="Columns names">
+        <DataListItem
+          id="groups-columns-header"
+          aria-labelledby="Columns names"
```

Comment:
The `aria-labelledby` attribute should reference an `id` of the element that labels this one. Otherwise an `aria-label` attribute should be used with a human readable string.

Reply:
Thanks @jonkoops I've just pushed a fix. Please let me know if that's better now.

Reply:
@agagancarczyk it still contains a human readable string in an attribute that is expected to be the ID of an element that describes it instead. See https://tink.uk/the-difference-between-aria-label-and-aria-labelledby/

Reply:
Thanks, I changed `aria-labelledby` to `aria-label` as it makes more sense in this case.